### PR TITLE
feat(new app command): implemented new app creation command from temp…

### PR DESCRIPTION
**Pull request by pgrzy-golem**
…lates

JST-424

### CODE REVIEW

File: data/project-templates/js-node/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "nodejs-golem-app",
+  "version": "1.0.0",
+  "description": "NodeJS script using Go",
```
Comment (mgordel) [inline data/project-templates/js-node/package.json:4]: this is probably a wrong description...?

File: data/project-templates/js-node/src/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import * as dotenv from "dotenv";
+import { LogLevel, ProposalFilters, TaskExecutor } from "@golem-sdk/golem-js";
+
+dotenv.config();
```
Comment (mgordel) [inline data/project-templates/js-node/src/index.js:4]: I wonder if this is necessary in this snippet? Or maybe you need to add a comment that you can put YAGNA_APPKEY in the .env or specify it in yagnaOptions...

File: data/project-templates/js-node/src/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    // Control the execution of tasks
+    maxTaskRetries: 0,
+
+    // Useful for debugging
+    logLevel: LogLevel.Info,
+    taskTimeout: 5 * 60 * 1000,
+  });
+
+  try {
+    // Your code goes here
```
Comment (mgordel) [inline data/project-templates/js-node/src/index.js:33]: maybe we should give an example of using `executor.run` and `ctx` ??

File: src/new/new.action.ts [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    description,
+    version,
+    author,
+  });
+
+  // TODO: Consider running npm install (or yarn, or whatever).
+
+  console.log(`Project created successfully in ${projectPath}.`);
+
+  if (!process.env.YAGNA_APPKEY) {
```
Comment (mgordel) [inline src/new/new.action.ts:164]: to consider whether to create a prompt for APPKEY and place it in .env

**Review by mgordel — APPROVED** [review]
Nothing blocking, just loose considerations..

File: data/project-templates/js-node/package.json (hunk 1 above)
Comment (pgrzy-golem) [reply data/project-templates/js-node/package.json:4]: Uncompleted :)
`Should be NodeJS script using Golem Network`

File: data/project-templates/js-node/src/index.js (hunk 2 above)
Comment (pgrzy-golem) [reply data/project-templates/js-node/src/index.js:4]: This is copied from the example supplied by Grzesiek

File: data/project-templates/js-node/src/index.js (hunk 3 above)
Comment (pgrzy-golem) [reply data/project-templates/js-node/src/index.js:33]: This is copied from the example supplied by Grzesiek - don't know if we should be making changes here

File: src/new/new.action.ts (hunk 4 above)
Comment (pgrzy-golem) [reply src/new/new.action.ts:164]: to be discussed with Grzesiek

**Pull request merged**